actionview/lib/action_view/lookup_context.rb

```diff
@@ -16,6 +16,8 @@ module ActionView
   # only once during the request, it speeds up all cache accesses.
   class LookupContext #:nodoc:
     attr_accessor :prefixes, :rendered_format
+    deprecate :rendered_format
+    deprecate :rendered_format=
```

Comment:
The LookupContext is private API, why bother deprecating the attribute reader/writer?

Reply:
Hehe, I'll ✂️.  No reason really.

Reply:
Actually I wanted to make 100% sure nobody else was calling it, and it looks like not

Reply:
Thanks for doing that @tenderlove! This is a method being used by the `rabl-rails` gem as we just noticed the warning on upgrading Rails. We'll work on an update to resolve the deprecation.

Is there an alternative to finding out the rendered format for an action?

Reply:
@javierjulio you should be able to get the format from the `rendered_template`. 62cb3ee35c5c0d729c54b057c37b8f6bed4dda7f may help. Let me know if I can help more!